### PR TITLE
1. directx-headers - make this its own package to facilitate reuse by…

### PR DESCRIPTION
**Pull request by JPeterMugaas**
… any other libraries.  Includes .pc file and *.cmake support.  Current version.

2. add directx-headers and polly as build-time dependencies.

### CODE REVIEW

**Comment by lazka** [comment]
@pal1000 any thoughts on this?

**Comment by pal1000** [comment]
LGTM, but FYI Mesa3D can actually use cmake to pick dependencies  via `-DCMAKE_PREFIX_PATH`, but just adding cmake as build dependency might be enough

File: mingw-w64-directx-headers/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+#depend upon CMake's findpackage ffunction.
+
+_realname=directx-headers
+pkgbase=mingw-w64-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+pkgver=1.610.0
+pkgrel=1
+pkgdesc="Official DirectX headers available under an open source license (mingw-w64)"
+arch=('any')
+mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64' 'clang32')
```
Comment (jeremyd2019) [inline mingw-w64-directx-headers/PKGBUILD:14]: Looks like 'clangarm64' was forgotten here, mesa build failed there due to lack of this package.

Comment (JPeterMugaas) [reply mingw-w64-directx-headers/PKGBUILD:14]: I have fixed that in another PR.  https://github.com/msys2/MINGW-packages/pull/17221

**Pull request merged**